### DISCUSSION TITLE
Other object detectors

### DISCUSSION THREAD

**Discussion by asdf2kr** [fredzzhang/upt#74] [Q&A] [answered]
Hi, @fredzzhang 
Thanks for your great work! 

I have a question. you trained upt using detr. 
I wonder if you've tried other transformer-based object detectors ? (like DeformableDETR). 
If you've tried, i want to know the performance.

Thank you

**Comment by fredzzhang** [comment, marked as answer]
Hi @asdf2kr,

Thanks for taking an interest in our work.

Yes we have tried other object detectors like the one you mentioned. Unfortunately, I cannot disclose the results at the moment, as it is still ongoing work. So, stay tuned!

Fred.

**Reply by asdf2kr** [reply]
Hi @fredzzhang 
Thank you for your answer.

I've tried other object detectors.
However I simply used hs, features from other detectors, I got a lower performance than DETR. (used same backbone)

Is this the right result?

(I understand that you can't mention the exact performance because it's still ongoing work. I just want to know if the performance decreases when i simply changed detector)

**Reply by fredzzhang** [reply]
In general, if the detector is stronger than DETR, you should be getting stronger HOI performance as well.

Fred.
